internal/cmd/skupper/token/kube/token_redeem.go

```diff
@@ -97,7 +64,7 @@ func (cmd *CmdTokenRedeem) ValidateInput(args []string) []error {
 	if siteList == nil || len(siteList.Items) == 0 {
 		validationErrors = append(validationErrors, fmt.Errorf("A site must exist in namespace %s before a token can be redeemed", cmd.namespace))
 	} else {
-		if !utils.SiteConfigured(siteList) {
+		if !utils2.SiteConfigured(siteList) {
```

Comment:
Again, we should check for site active.   But can be addressed later.    Just add a TBD in code 
(I missed this in initial implementation)

Reply:
Issue created: #1614